sycl/test-e2e/DeviceLib/math_fp64_test.cpp

```diff
@@ -61,6 +61,7 @@ void device_math_test(s::queue &deviceQueue) {
         double subnormal;
         *((uint64_t *)&subnormal) = 0xFFFFFFFFFFFFFULL;
 
+        res_access[i++] = rint(0.9);
         res_access[i++] = cos(0.0);
```

Comment:
Could you please test `pow(1.0, 1.0f);`? `pow(double, float)` does not handled correctly in my environment.
Probably all multi-parameter functions aren't intercepted if one of parameters requires a conversion.